planner/core/testdata/plan_suite_in.json

```diff
@@ -1186,5 +1186,22 @@
       "select a, count(*) from t group by a -- shouldn't be rewritten",
       "select sum(a) from t -- sum shouldn't be rewritten"
     ]
+  },
+  {
+    // Test redundant condition in select.
```

Comment:
We usually add comment in test case function of planner/core/predicate_simplification_test.go, not here.